src/test/java/com/amazon/opendistroforelasticsearch/search/asynchronous/utils/RestTestUtils.java

```diff
@@ -73,7 +73,7 @@ public static Request buildHttpRequest(SubmitAsynchronousSearchRequest submitAsy
         SearchRequest searchRequest = submitAsynchronousSearchRequest.getSearchRequest();
         Request request = new Request(HttpPost.METHOD_NAME,
                 /*trim first backslash*/
-                endpoint(searchRequest.indices(), AsynchronousSearchPlugin.BASE_URI.substring(1)));
+                endpoint(searchRequest.indices(), AsynchronousSearchPlugin.OPENSEARCH_BASE_URI.substring(1)));
```

Comment:
should we do a random between the endpoints to ensure both of them work in the meantime

Reply:
I think you want a couple of predictable tests for backward compatibility instead.